.github/workflows/ci.yml

```diff
@@ -195,8 +195,10 @@ jobs:
           git config --global user.name 'The Miri Conjob Bot'
           git config --global user.email 'miri@cron.bot'
       - name: get changes from rustc
+        shell: bash
         run: ./miri rustc-pull
```

Comment:
I think the issue is that the Miri repo is not checked out in this job.

Also, don't you have to install rustup-toolchain-install-master somewhere?

Reply:
force pushed a change that should do everything needed